File: C#/1014 - Consumo.cs
using System;

namespace Consumo
{
    class Program
    {
        static void Main(string[] args) {
            int distancia;
            double combustivel, gasto;

            distancia = Convert.ToInt32(Console.ReadLine());
            combustivel = Convert.ToDouble(Console.ReadLine());

            gasto = distancia / combustivel;

            Console.WriteLine(string.Format($"{gasto.ToString("0.000")} km/l"));
        }
    }
}

File: Java/1014 - Consumo.java
import java.io.IOException;
import java.util.*;

public class Main{
    public static void main(String[] args){
        int distancia;
        double combustivel, gasto;
        
        Scanner valor = new Scanner(System.in);
        distancia = valor.nextInt();
        combustivel = valor.nextDouble();
        
        gasto = distancia / combustivel;
        
        System.out.println(String.format("%.3f km/l", gasto));
    }
}
